database/pgx/pgx.go

```diff
@@ -28,6 +28,7 @@ import (
 func init() {
 	db := Postgres{}
 	database.Register("pgx", &db)
+	database.Register("pgx/v4", &db)
```

Comment:
Register this as `pgx4` since `pgx/v4` is verbose. We don't need to be consistent with the upstream. Also, our DSN/connection string parsing uses URL parsing and `/` is not a [valid scheme character](https://datatracker.ietf.org/doc/html/rfc3986#section-3.1)